src/main/java/uk/gov/justice/hmpps/prison/api/resource/ServiceAgencySwitchesResource.kt

```diff
@@ -55,13 +53,12 @@ class ServiceAgencySwitchesResource(private val service: ServiceAgencySwitchesSe
   @PreAuthorize("hasRole('SERVICE_AGENCY_SWITCHES')")
   @PostMapping("/{serviceCode}/prison/{prisonId}")
   @ResponseStatus(HttpStatus.CREATED)
-  @Hidden // TODO SDIT-797 Remove annotation once tables EXTERNAL_SERVICES  and SERVICE_AGENCY_SWITCHES are live
   fun addServicePrison(
     @PathVariable
-    @Parameter(name = "The code of the service from the EXTERNAL_SERVICES table")
+    @Parameter(description = "The code of the service from the EXTERNAL_SERVICES table")
```

Comment:
This caused the Open API tests to fail - not sure why they just started failing but clearly needed fixing.